CrossDisciplinary-1/UT-CrossDisciplinary-1.ifc

```diff
@@ -0,0 +1,49 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [Ifc4X3NotAssigned]'), '2;1');
+FILE_NAME ('UT-CrossDisciplinary-1.ifc', '2021-02-08T17:15:49', (), (), 'ACCA_IFC.dll Version 500', 'ACCA IfcInfraGenerator', $);
+FILE_SCHEMA (('IFC4X3_RC3'));
+ENDSEC;
+DATA;
+#1 = IFCOWNERHISTORY(#2, #6, $, .NOCHANGE., $, $, $, 1612804549);
+#2 = IFCPERSONANDORGANIZATION(#3, #4, $);
+#3 = IFCPERSON($, 'Cianciulli', 'Michelangelo', $, $, $, $, $);
+#4 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#5 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#6 = IFCAPPLICATION(#5, '100', 'ACCA IfcInfraGenerator', 'ACCA IfcInfraGenerator');
+#7 = IFCPROJECT('3TkAlDBHHF$xq2dgZCH3rI', #1, 'Default Project', $, $, $, $, (#16), #8);
+#8 = IFCUNITASSIGNMENT((#9, #10, #11, #12));
+#9 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#10 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#11 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#12 = IFCCONVERSIONBASEDUNIT(#13, .PLANEANGLEUNIT., 'DEGREE', #14);
+#13 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
+#14 = IFCMEASUREWITHUNIT(IFCRATIOMEASURE(1.74532925E-2), #15);
+#15 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#16 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-4, #17, #19);
+#17 = IFCAXIS2PLACEMENT3D(#18, $, $);
+#18 = IFCCARTESIANPOINT((0., 0., 0.));
+#19 = IFCDIRECTION((0., 1.));
+#20 = IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body', 'Model', *, *, *, *, #16, $, .MODEL_VIEW., $);
+#21 = IFCSITE('3t2PewUcHA5fz9fmoiszqp', #1, 'Default Site', $, $, #22, $, $, .ELEMENT., (0, 0, 0), (0, 0, 0), 0., $, $);
+#22 = IFCLOCALPLACEMENT($, #25);
+#23 = IFCDIRECTION((0., 0., 1.));
+#24 = IFCDIRECTION((1., 0., 0.));
+#25 = IFCAXIS2PLACEMENT3D(#18, #23, #24);
+#26 = IFCROAD('0DN9cEmCPFUgDDXQbLxZvD', #1, 'a road', $, $, $, $, $, .ELEMENT.);
+#27 = IFCFACILITYPART('0yFg1D40940PDUfPFz$nBF', #1, 'a road part', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.ROADSEGMENT.), .LONGITUDINAL.);
+#28 = IFCFACILITYPART('3r4b0myeL6UA$c7cgEphKD', #1, 'a road part lateral, left', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.ROADSEGMENT.), .LATERAL.);
+#29 = IFCFACILITYPART('3xh0isp$n0MvRhHMOZm3Gk', #1, 'a road part lateral, right', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.ROADSEGMENT.), .LATERAL.);

```

Comment:
ROADSEGMENT is not usually associated with a Lateral facility part, it is used to breakup the longitudinal lengh of the road, the more logical value would be say a TRAFFICLANE. see summary comment for further details

Reply:
removed the lateral facility parts as to keep the UT simpler and semantically correct

Reply:
All Good